src/Compilers/Test/Core/CompilationVerifier.cs

```diff
@@ -229,6 +229,55 @@ public void VerifyTypeIL(string typeName, Action<string> validateExpected)
             validateExpected(output.ToString());
         }
 
+        public void VerifyMethodIL(string typeName, string methodName, string expected)
+        {
+            VerifyMethodIL(typeName, methodName, output => AssertEx.AssertEqualToleratingWhitespaceDifferences(expected, output, escapeQuotes: false));
+        }
+
+        public void VerifyMethodIL(string typeName, string methodName, Action<string> validateExpected)
```

Comment:
>VerifyMethodIL [](http://example.com/codeflow?start=20&length=14)

Why do we need this helper? Shouldn't ```VerifyIL``` just work? Also, it looks like this helper is going to emit again, making testing slower.

Reply:
> It's used to verify method's IL signature which VerifyIL doesn't include. 

We usually do not use IL to verify signatures. We have APIs for to check particular aspects.